USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -8,10 +8,32 @@ namespace USFMToolsSharp
     public class HtmlRenderer
     {
         public List<string> UnrenderableTags;
+        public List<string> FootnoteTextTags;
 
+        private bool isSingleSpaced = true;
+        private bool hasOneColumn = true;
+        private bool isL2RDirection = true;
+        private bool isTextJustified = false;
+        private bool separateChapters = false;
+
+        private string InsertedHTML = "";
+        private string InsertedFooter = "";
         public HtmlRenderer()
         {
             UnrenderableTags = new List<string>();
+            FootnoteTextTags = new List<string>();
+        }
+        public HtmlRenderer(bool isSingleSpaced, bool hasOneColumn, bool isL2RDirection, bool isTextJustified, bool separateChapters)

```

Comment:
You can set default values for these so that people don't have to specify all of them. Makes it more usable. So for example
`public HtmlRederer(bool isSingleSpaced = true) that way people could selectively include only the things they care about and leave others as defaults. This also prevents someone from needing to have knowledge as to what the defaults are.